src/main/interface/msp.c

```diff
@@ -523,10 +523,11 @@ static bool mspCommonProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProce
         break;
 
     case MSP_ANALOG:
-        sbufWriteU8(dst, (uint8_t)constrain(getBatteryVoltage(), 0, 255));
+        sbufWriteU8(dst, (uint8_t)constrain((getBatteryVoltage() + 5) / 10, 0, 255));
```

Comment:
You're using ```(getBatteryVoltage() + 5) / 10``` multiple times throughout this PR, make it a macro or a function?
As I understand it, this is a math round function?

Reply:
Yes, it's a round function. I could do a new function in battery.c as I've done before but then switched to this.
@mikeller What do you think?

Reply:
@fgiudice98: In this case, introducing a `getLegacyBatteryVoltage()` probably makes sense.

Reply:
Cool, sounds good.